### ISSUE TITLE
statName and statDescription missing in stats definition

### ISSUE THREAD

**Issue by jcsilvers** [Bungie-net/api#268]
All of the medals pulled from https://www.bungie.net/Platform/Destiny2/Stats/Definition/ are missing statName and statDescrption. 

Current data for each medal is shaped like below:

"statName": "###medal_strings.pvp_ability_dawnblade_aerial###",
"statDescription": "###medal_descriptions.Medals_pvp_medal_ability_dawnblade_aerial###",

I would expect the statName to be a formatted for display and the statDescription should detail what the requirements are to achieve.

**Comment by maciek134** [comment]
Same in the manifest sqlite files, along with about half of the ActivityMode definitions.

**Comment by Tetron-bng** [comment]
Currently we don't provide human readable strings for all of our stats. Only the ones Bungie.net and the companion apps actually displays.

**Comment by enderjs** [comment]
Thanks for the replay. Will you be adding them in the future?

**Comment by Tetron-bng** [comment]
Definitely adding medals at some point. There were many counters throughout D1 that never got names and descriptions.  Everything we add needs to be localized into 8 other languages, so we tend to conserve resources on data we don't display ourselves.

**Comment by sgtfrankieboy** [comment]
Aren't the names for Medals already localized as they are shown in-game?

**Comment by Tetron-bng** [comment]
Yes, medals are the exception. We need to do work to import that data but it is low priority at the moment.

**Comment by vpzed** [comment]
Could we get an unofficial "lookup table" for what the game calls a given statId so we could integrate it into our apps?  There are a lot of those "###somthing###" names and I don't know how to get them even manually.

**Comment by Tetron-bng** [comment]
The game does not show many (most) of the stats the API reveals.  The stat id itself is what we call the stat internally and should give you a good idea what it represents.  As we begin to expose the stat on our apps and website, you will see localized display names come through, but that will always be a small subset of what is tracked and available.

**Comment by vpzed** [comment]
Those medals pop up in PvP all the time (or used to in D1, I haven't played much PvP in D2).  But point taken.  I'll just stop pulling Medals data because I can't display it.

**Comment by v-fedorov** [comment]
If anyone is interested I have a spreadsheet open with the corrected values.
Feel free to update any missing ones at the bottom if you know them 

https://docs.google.com/spreadsheets/d/16MbZayyb0QSzAoL029ZaOq0qdcLlrb196U1V9DuS9rs/edit?usp=sharing

**Comment by vpzed** [comment]
I added a new tab to the spreadsheet with a complete list as of this manifest.  I added a new column with manually created statNames that include the medal names people got from watching Twitch streams and I added detail to the manifest statNames so they are unique.  It's a first draft but for now it might be usable for a lookup-up table to display historical stats in an app or database.
